node/pkg/watchers/evm/chain_config_test.go

```diff
@@ -128,6 +128,7 @@ func getFinalityForTest(env common.Environment, chainID vaa.ChainID) (instantFin
 		chainID == vaa.ChainIDArbitrumSepolia ||
 		chainID == vaa.ChainIDBase ||
 		chainID == vaa.ChainIDBaseSepolia ||
+		chainID == vaa.ChainIDBerachain || // Berachain really isn't instant finality, despite what this doc says: https://docs.berachain.com/faq/
```

Comment:
I don't think this comment is accurate. Instead, it's just that the chain config will require Berachain's blocks to have a `finalized` status.

Reply:
Close enough, especially since this function will be deleted soon anyway (per the function prologue).